backend/schemas.py

```
@@ -109,3 +109,57 @@ class UserUpdate(BaseModel):
+class OrderCreate(BaseModel):  
+    user_id: int  
+    status: str  
+    order_items: List[OrderItemCreate]  
+    bill: BillCreate  
+  
+class OrderOut(BaseModel):  
+    order_id: int  
+    user_id: int  
+    status: str  
+    created_at: datetime  
+    order_items: List[OrderItemOut]  
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)